mesa/Person.py

```diff
@@ -168,16 +169,15 @@ def makeSchoolLinks(self, asker):
             person.neighbors.get("school").add(self)
 
     def addCriminalLink(self, asker):
-        #todo: Links between people do not have the "criminal_link_weight" attribute
-        weight = self.criminal_link_weight.get(asker)
-        if weight == None:
-            self.neighbors.get("criminal").add(asker)
-            self.criminal_link_weight[asker] = 1
-            asker.neighbors.get("criminal").add(self)
-            asker.criminal_link_weight[asker] = 1
-        else:
-            self.criminal_link_weight[asker]  += 1
-            asker.criminal_link_weight[asker] += 1 
+        """
+        Create a two-way criminal links in-place and update the connection weight
+        :param asker: agent
+        :return: None
+        """
+        self.neighbors.get("criminal").add(asker)
+        self.num_co_offenses[asker] = 1
```

Comment:
It should not happen, but what if the link exists? I would play safe and use
        if not self.num_co_offenses[asker]: self.num_co_offenses[asker] = 1

Reply:
It shouldn't happen but this is a premature optimization attempt of mine, I'll add a check!